mlrun/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -575,9 +486,112 @@ def list_model_endpoints(
 
         return endpoint_list
 
+    def verify_project_has_no_model_endpoints(self, project_name: str):
+        """This method is usually being used during the process of deleting a project. It verifies that there no
```

Comment:
Refrase - docstring for function should not start with where it is called from:
```suggestion
        """Verifies that there are no model endpoint records in the DB by trying to list all of the project model endpoints."""
```